integration/dubbo-alibaba/src/main/java/io/seata/integration/dubbo/alibaba/AlibabaDubboTransactionPropagationFilter.java

```diff
@@ -0,0 +1,105 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.integration.dubbo.alibaba;
+
+import com.alibaba.dubbo.common.extension.Activate;
+import com.alibaba.dubbo.rpc.Filter;
+import com.alibaba.dubbo.rpc.Invocation;
+import com.alibaba.dubbo.rpc.Invoker;
+import com.alibaba.dubbo.rpc.Result;
+import com.alibaba.dubbo.rpc.RpcContext;
+import com.alibaba.dubbo.rpc.RpcException;
+import io.seata.core.context.RootContext;
+import io.seata.integration.dubbo.DubboConstants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The type Transaction propagation filter.
+ *
+ * @author sharajava
+ */
+@Activate(group = {DubboConstants.PROVIDER, DubboConstants.CONSUMER}, order = 100)
+public class AlibabaDubboTransactionPropagationFilter implements Filter {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AlibabaDubboTransactionPropagationFilter.class);
+
+    @Override
+    public Result invoke(Invoker<?> invoker, Invocation invocation) throws RpcException {
+        if (DubboConstants.ALIBABADUBBO) {
```

Comment:
Because of dependence on seata-dubbo (apache-dubbo), is it alibaba-dubbo?

Reply:
> Because of dependence on seata-dubbo (apache-dubbo), is it alibaba-dubbo?

used to determine whether it is Alibaba or Apache @slievrly

Reply:
if (!DubboConstants.ALIBABADUBBO) {return invoker.invoke(invocation);}
reduce original changes

Reply:
> if (!DubboConstants.ALIBABADUBBO) {return invoker.invoke(invocation);}
> reduce original changes

get it